releases/release-5.1.4.md

```diff
@@ -0,0 +1,167 @@
+---
+title: TiDB 5.1.4 Release Notes
+category: Releases
+---
+
+# TiDB 5.1.4 Release Notes
+
+Release Date: xx, 2022
+
+TiDB version: 5.1.4
+
+## Compatibility changes
+
+## Feature enhancements
+
+## __unsorted
+
++ TiDB
+
+    (dup) - Fix wrong results of the `microsecond` function in vectorized expressions [#29244](https://github.com/pingcap/tidb/issues/29244)
+    - Set default value of tidb_analyze_version to 1 in v5.1 and v5.2 [#31748](https://github.com/pingcap/tidb/issues/31748)
+    - Fix a panic that may happen when using `on duplicate key update`. [#28078](https://github.com/pingcap/tidb/issues/28078)
+    - Fix `MaxDays` and `MaxBackups` not working for slow log. [#25716](https://github.com/pingcap/tidb/issues/25716)
+    - Fix an issue that adding index panics by chance. [#27687](https://github.com/pingcap/tidb/issues/27687)
+    - Fix wrong result for join with enum type [#27831](https://github.com/pingcap/tidb/issues/27831)
+    (dup) - Fix the panic when using the `CASE WHEN` function on the `ENUM` data type [#29357](https://github.com/pingcap/tidb/issues/29357)
+    - Fix a memory leak bug when using @@tidb_analyze_version = 2 [#29305](https://github.com/pingcap/tidb/pull/29305)
```

Comment:
```suggestion
    - Fix a memory leak bug when using @@tidb_analyze_version = 2 [#29305](https://github.com/pingcap/tidb/issues/3776)
```

Reply:
'On Call #3776' is about the oncall, it is nothing related to Github Issue #3776.

Reply:
@tiancaiamao Could you help provide the issue link for https://github.com/pingcap/tidb/pull/29305. Thanks.